### PR TITLE
Docker branch update

### PR DESCRIPTION
**Pull request by apeixinho**

### CODE REVIEW

File: backend/src/main/java/com/bfwg/service/impl/AuthorityServiceImpl.java [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
+    private final AuthorityRepository authorityRepository;
+
+    @Autowired
+    public AuthorityServiceImpl(AuthorityRepository authorityRepository) {
+        this.authorityRepository = authorityRepository;
+    }
+
+    @Override
+    public List<Authority> findById(Long id) {
+        // TODO Auto-generated method stub
```
Comment (bfwg) [inline backend/src/main/java/com/bfwg/service/impl/AuthorityServiceImpl.java]: Can we remove this dev comment?

File: backend/src/main/java/com/bfwg/service/impl/AuthorityServiceImpl.java [hunk 2]
```diff
@@ -1,34 +1,42 @@
...
+
+        Authority auth = this.authorityRepository.getOne(id);
+        List<Authority> auths = new ArrayList<>();
+        auths.add(auth);
+        return auths;
+    }
+
+    @Override
+    public List<Authority> findByName(UserRoleName name) {
+        // TODO Auto-generated method stub
```
Comment (bfwg) [inline backend/src/main/java/com/bfwg/service/impl/AuthorityServiceImpl.java]: This one as well. Can we remove please?

**Review by bfwg — COMMENTED** [review]
Amazing work @apeixinho , what are the TODO comments for?

File: backend/src/main/java/com/bfwg/service/impl/AuthorityServiceImpl.java (hunk 1 above)
Comment (apeixinho) [reply backend/src/main/java/com/bfwg/service/impl/AuthorityServiceImpl.java]: Done

File: backend/src/main/java/com/bfwg/service/impl/AuthorityServiceImpl.java (hunk 2 above)
Comment (apeixinho) [reply backend/src/main/java/com/bfwg/service/impl/AuthorityServiceImpl.java]: Done

Comment (apeixinho) [reply backend/src/main/java/com/bfwg/service/impl/AuthorityServiceImpl.java]: I understand this is a huge pull request. Which is not the best way ...
But I had so many changes is this branch that I forgot ... long story, short, I still pushed the pull request button.
Take your time, no worries

**Comment by bfwg** [comment]
LGTM 👍 Thanks for the effort.

**Pull request merged**